src/react-native/11-navigation/navigation.md

```diff
@@ -107,10 +107,16 @@ The `Tab.Screen` component takes a `name` prop that is used to identify the scre
 
 ### Setup 1
 
-✏️ Install the `@react-navigation/native` and `@react-navigation/bottom-tabs` packages:
+✏️ Install the `@react-navigation/native`, `@react-navigation/bottom-tabs`, and `@react-native-safe-area-context` packages:
 
 ```bash
-npm install @react-navigation/native @react-navigation/bottom-tabs
+npm install @react-navigation/native@6 npm install @react-navigation/bottom-tabs@6 npm install react-native-safe-area-context@4
```

Comment:
This command needs to be fixed (`npm install` shouldn’t be repeated over and over).